firmware_tools/ghidra/vxhunter_utility/symbol.py

```diff
@@ -253,77 +242,18 @@
 vx_5_netPool.replaceAtOffset(0x18, int_type, 4, "clSizeMax", "maximum cluster size")
 vx_5_netPool.replaceAtOffset(0x1C, int_type, 4, "clLg2Min", "cluster log2 minimum size")
 vx_5_netPool.replaceAtOffset(0x20, int_type, 4, "clSizeMin", "minimum cluster size")
-vx_5_netPool.replaceAtOffset(0x24, vx_5_clTbl_array_data_type, vx_5_clTbl_array_data_type.getLength(),
-                             "clTbl", "pool table")
+vx_5_netPool.replaceAtOffset(0x24, vx_5_clTbl_array_data_type, vx_5_clTbl_array_data_type.getLength(), "clTbl", "pool table")
 vx_5_netPool.replaceAtOffset(0x50, void_ptr_type, 4, "pPoolStat", "pool statistics")
 vx_5_netPool.replaceAtOffset(0x54, void_ptr_type, 4, "pFuncTbl", "ptr to function ptr table")
 
-
 function_name_chaset = string.letters
 function_name_chaset += string.digits
 function_name_chaset += "_:.<>,*"  # For C++
 function_name_chaset += "()~+-=/%"  # For C++ special eg operator+(ZafBignumData const &,long)
 ghidra_builtin_types = [
-    'undefined',
-    'byte',
-    'uint',
-    'ushort',
-    'bool',
-    'complex16',
-    'complex32',
-    'complex8',
-    'doublecomplex',
-    'dwfenc',
-    'dword',
-    'filetime',
-    'float10',
-    'float16',
-    'float2',
-    'float4',
-    'float8',
-    'floatcomplex',
-    'guid',
-    'imagebaseoffset32',
-    'imagebaseoffset64',
-    'int16',
-    'int3',
-    'int5',
-    'int6',
-    'int7',
-    'long',
-    'longdouble',
-    'longdoublecomplex',
-    'longlong',
-    'mactime',
-    'prel31',
-    'qword',
-    'sbyte',
-    'schar',
-    'sdword',
-    'segmentedcodeaddress',
-    'shiftedaddress',
-    'sqword',
-    'sword',
-    'wchar16',
-    'wchar32',
-    'uchar',
-    'uint16',
-    'uint3',
-    'uint5',
-    'uint6',
-    'uint7',
-    'ulong',
-    'ulonglong',
-    'undefined1',
-    'undefined2',
-    'undefined3',
-    'undefined4',
-    'undefined5',
-    'undefined6',
-    'undefined7',
-    'undefined8',
-    'wchar_t',
-    'word'
+    'undefined', 'byte', 'uint', 'ushort', 'bool', 'complex16', 'complex32', 'complex8', 'doublecomplex', 'dwfenc', 'dword', 'filetime', 'float10', 'float16', 'float2', 'float4', 'float8', 'floatcomplex', 'guid', 'imagebaseoffset32', 'imagebaseoffset64',
+    'int16', 'int3', 'int5', 'int6', 'int7', 'long', 'longdouble', 'longdoublecomplex', 'longlong', 'mactime', 'prel31', 'qword', 'sbyte', 'schar', 'sdword', 'segmentedcodeaddress', 'shiftedaddress', 'sqword', 'sword', 'wchar16', 'wchar32', 'uchar',
+    'uint16', 'uint3', 'uint5', 'uint6', 'uint7', 'ulong', 'ulonglong', 'undefined1', 'undefined2', 'undefined3', 'undefined4', 'undefined5', 'undefined6', 'undefined7', 'undefined8', 'wchar_t', 'word'
```

Comment:
Please don't edit this kind of code, newline makes these types easier to add or delete.

Reply:
My apologies -- I have returned the formatting to its original state, and also alphabetized the list to make it easier to find entries when you wish to add or delete them.